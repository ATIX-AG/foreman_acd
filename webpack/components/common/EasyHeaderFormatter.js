import React from 'react';
import PropTypes from 'prop-types';
import {
  Table,
} from 'patternfly-react';

const EasyHeaderFormatter = (value, { column }) => {
  return
  (
    <Table.Heading aria-label={column.header.label} {...column.header.props}>
    {value}
    </Table.Heading>
  );
};

EasyHeaderFormatter.propTypes = {
  value: PropTypes.string.isRequired,
  column: PropTypes.object.isRequired,
};

export default EasyHeaderFormatter;
