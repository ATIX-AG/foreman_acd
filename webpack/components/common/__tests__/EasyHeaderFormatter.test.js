import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import EasyHeaderFormatter from '../EasyHeaderFormatter';

const noop = () => {};

const value = 'Hallo Welt';
const data = {
  header: {
    label: 'MyLabel',
    props: 0
  }
}

const fixtures = {
  'should render add parameter': {
    value,
    column: data,
  },
};

describe('EasyHeaderFormatter', () =>
  testComponentSnapshotsWithFixtures(EasyHeaderFormatter, fixtures));
