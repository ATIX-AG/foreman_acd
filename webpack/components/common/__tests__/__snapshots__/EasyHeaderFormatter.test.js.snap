// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EasyHeaderFormatter should render add parameter 1`] = `""`;
